src/FsAutoComplete.Core/Lexer.fs

```diff
@@ -48,7 +48,7 @@ module Lexer =
       |> Seq.choose (fun s -> if s.StartsWith "--define:" then Some s.[9..] else None)
       |> Seq.toList
 
-    let sourceTokenizer = FSharpSourceTokenizer(defines, Some "/tmp.fsx")
+    let sourceTokenizer = FSharpSourceTokenizer(defines, Some "/tmp.fsx", None)
```

Comment:
we should check if we have langVersion available via the project options - we should try to provide it here if possible. This can be a follow-up issue if you don't want to tackle it.

Reply:
I think I got it.